code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -624,6 +625,8 @@
 			intensity += R.intensitymod * R.volume
 			duration += R.durationmod * R.volume
 			radius += R.radiusmod * R.volume
+			if(R.fire_penetrating && R.volume >= 10)
```

Comment:
use define for 10

Reply:
Would it be good to also add defines for some of the old code like for phoron requiring 10u and phosphorus having a radius of volume/10?

Reply:
yeah